docs/details/commands/pmm-admin.md

```diff
@@ -364,6 +407,10 @@ If `--stats-collections=db1,db2.col1` then the collectors are run as follows:
 |db1| All the collections|
 |db2| **Only** for collection col1|
 
+Enable all collectors and limit monitoring for the `dbstats` and `indexstats` to all collections in `db1` and for `col1` collection in `db2`, no limit for number of collection in `db1`:
```

Comment:
why only dbstats and indexstats, this example applies for collecting all collectors, there isn't any disable-collectors flag used here.

Reply:
@JiriCtvrtka could you please confirm ?

Reply:
`pmm-admin add mongodb --username=admin --password=admin_pass --enable-all-collectors --max-collections-limit=0 --disable-collectors=collstats mongodb_srv_1 127.0.0.1:27017`

If `--stats-collections=db1,db2.col1` then the collectors are run as follows:

|Database|Collector is run on|
|-----|-----|
|db1| All the collections|
|db2| **Only** for collection col1|

Enable all collectors and limit monitoring for `dbstats` and `indexstats` to all collections in `db1` and `col1` collection in `db2`, without limiting `max-collections-limit` for a number of collections in `db1`:

I think it should be:
Enable all collectors and limit monitoring for `dbstats` and `indexstats` and `topmetrics` to all collections in `db1` and `col1` collection in `db2`, without limiting `max-collections-limit` for a number of collections in `db1`: